### PR TITLE
Removed _old lines from schema

### PR DESCRIPTION
**Pull request by icanberk**

### CODE REVIEW

**Comment by icanberk** [comment]
@dlbucci how do I test this?

**Comment by icanberk** [comment]
Resolves #43

**Comment by icanberk** [comment]
So I tested this by creating a db, migrating it and populating it. Everything seems to work well and we don't have the _old columns in the db anymore. I think this is good to go.

File: db/schema.rb [hunk 1]
```diff
@@ -148,7 +144,6 @@
     t.boolean  "dropped",              limit: 1,   default: false
     t.string   "nickname",             limit: 255
     t.boolean  "course_assistant",     limit: 1,   default: false
-    t.float    "tweak_old",            limit: 24,  default: 0.0
     t.boolean  "absolute_tweak",       limit: 1,   default: true
```
Comment (dlbucci) [inline db/schema.rb]: I believe this (absolute_tweak) can also be removed.

**Comment by dlbucci** [comment]
Isn't there a migration file for this?  I don't think just changing the schema will affect the database.

**Comment by icanberk** [comment]
Hmm.. this actually seems to be the wrong practice.
http://stackoverflow.com/questions/11301009/is-it-considered-safe-to-manually-edit-schema-rb-in-rails

I will just migrations that will remove these keys.

(Rails is so weird)

**Comment by icanberk** [comment]
@dlbucci this should be good to go now

**Comment by icanberk** [comment]
Fyi, schema.rb is autogenerated. I'm not sure why it diffed a lot of those lines.

**Comment by dlbucci** [comment]
The _backup columns in CUD can also be removed.  There's another absolute_tweak column that I believe can also be removed.

**Comment by icanberk** [comment]
Are you sure we are not using submission.absolute_tweak  ?

**Comment by dlbucci** [comment]
And the feedback_file columns in the scores table can also be removed, since we don't really use that anymore, although maybe check that we aren't.

**Comment by dlbucci** [comment]
I think tweaks have been moved into the Tweak model, so the column is no longer used, but you can check to be sure.

**Comment by icanberk** [comment]
I see. It doesn't seem to be used anywhere.

`feedback_file` seems to be in use btw.

**Comment by dlbucci** [comment]
Also, absolute_tweak is only mentioned in the schema and old migrations.

**Comment by icanberk** [comment]
Done

**Comment by dlbucci** [comment]
Code is there to handle feedback_file values, but I don't think that column can be set anywhere in the code.  If you want, we can leave it in for now, and remove it after a check.

**Comment by icanberk** [comment]
Yeah, let's not touch it for now.

**Comment by dlbucci** [comment]
And you've made sure this migration will run with the new additions?  Because we absolutely do not want a migration to fail in production.

**Comment by icanberk** [comment]
Yep

**Pull request merged**